src/components/ProviderWrapper/index.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { BrowserRouter } from "react-router-dom";
+import { AuthProvider } from "../../context/AuthContext";
+import { CategoryProvider } from "../../context/CategoryContext";
+import { QuestionProvider } from "../../context/QuestionContext";
+import { QuizProvider } from "../../context/QuizContext";
+import { ToastProvider } from "../../context/ToastContext";
+
+const ProviderWrapper = ({ children }) => {
+  return (
+    <>
```

Comment:
<BrowserRouter> is already a container tag, you don't require this to wrap with Fragment here

Reply:
You are right